app/styles/app.scss

```diff
@@ -1337,6 +1337,17 @@ nav.preprint-navbar {
         margin-right: 16px;
     }
 
+    // Hacks for very small preprints windows breaking navbar into two lines
+    // Don't display the preprint word on small screens and force the margin of the button over a bit
```

Comment:
@alexschiller could you create a ticket for fixing the navbar spacing in osf-style and add the link to the comment? That way if anyone is looking at this they will know whether it is say to remove the hack.